### PR TITLE
Box storage: Improvements from generator

### PR DESCRIPTION
**Pull request by jasonpaulos**
Child PR to #341 

This PR brings changes from https://github.com/algorand/generator/pull/43

It's not technically necessary, but it does improve the ergonomics of the box endpoints.

For some reason the code generation changes introduced many inconsequential newlines. The actual changeset is much smaller than it appears.

### CODE REVIEW

File: client/v2/algod/accountInformation.go [hunk 1]
```diff
@@ -35,6 +35,7 @@ type AccountInformation struct {
 // `none`.
 func (s *AccountInformation) Exclude(Exclude string) *AccountInformation {
 	s.p.Exclude = Exclude
+
```
Comment (tzaffi) [inline client/v2/algod/accountInformation.go:38]: :+1

**Review by tzaffi — COMMENTED** [review]
These changes look good except that I'd like to understand why the `BlockHash` methods are getting generated.

File: client/v2/algod/getBlockHash.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import (
+	"context"
+	"fmt"
+
+	"github.com/algorand/go-algorand-sdk/client/v2/common"
+	"github.com/algorand/go-algorand-sdk/client/v2/common/models"
+)
+
+// GetBlockHash get the block hash for the block on the given round.
+type GetBlockHash struct {
```
[the remark this answers is not in the document]
Comment (michaeldiamant) [reply client/v2/algod/getBlockHash.go:12]: @tzaffi It implies that REST API changes were merged upstream, but not re-generated in the SDK.  I didn't track down why `develop` wasn't updated.  In principle, it ought to be updated prior to merging the PR.

[the remark this answers is not in the document]
Comment (jasonpaulos) [reply client/v2/algod/getBlockHash.go]: The `develop` branch now has this change, and it's been merged into the PR branch, so this diff no longer shows.

**Review by michaeldiamant — APPROVED** [review]
I confirmed _only_ newline changes happen against `develop`, so I am comfortable merging this PR without having 1st brought the changes into `develop`.

I visually confirmed required properties set as expected + no unintended generated changes.

**Pull request merged**